gui-webview/routes/webhooks.js

```diff
@@ -60,19 +60,21 @@ router.post('/', (req, res) => {
     // There may be multiple if batched
     data.entry.forEach((pageEntry) => {
       // Iterate over each messaging event and handle accordingly
-      pageEntry.messaging.forEach((messagingEvent) => {
-        console.log({messagingEvent});
-        if (messagingEvent.message) {
-          receiveApi.handleReceiveMessage(messagingEvent);
-        } if (messagingEvent.postback) {
-          receiveApi.handleReceivePostback(messagingEvent);
-        } else {
-          console.log(
-            'Webhook received unknown messagingEvent: ',
-            messagingEvent
-          );
-        }
-      });
+      if(pageEntry.messaging){
```

Comment:
yes, you can use  `if (!pageEntry.messaging) { return; }`
and i think your alternative has the same effect with cleaner code.